modules/images/dominant-color/load.php

```diff
@@ -339,3 +320,69 @@ function wp_get_attachment_file_path( $attachment_id, $size = 'medium' ) {
 
 	return $filepath;
 }
+
+/**
+ * Gets the dominant color for an image attachment.
+ *
+ * @since n.e.x.t
+ *
+ * @param int $attachment_id Attachment ID for image.
+ * @return string|null Hex value of dominant color or null if not set.
+ */
+function dominant_color_get_dominant_color( $attachment_id ) {
+	if ( ! wp_attachment_is_image( $attachment_id ) ) {
+		return null;
+	}
+	$image_meta = wp_get_attachment_metadata( $attachment_id );
+	if ( ! is_array( $image_meta ) ) {
+		return null;
+	}
+
+	if ( ! isset( $image_meta['dominant_color'] ) ) {
+		return null;
+	}
+
+	return $image_meta['dominant_color'];
+}
+
+/**
+ * Returns whether an image attachment has transparency.
+ *
+ * @since n.e.x.t
+ *
+ * @param int $attachment_id Attachment ID for image.
+ * @return bool|null Whether the image has transparency, or null if not set.
+ */
+function dominant_color_has_transparency( $attachment_id ) {
+	$image_meta = wp_get_attachment_metadata( $attachment_id );
+	if ( ! is_array( $image_meta ) ) {
+		return null;
+	}
+
+	if ( ! isset( $image_meta['has_transparency'] ) ) {
+		return null;
+	}
+
+	return $image_meta['has_transparency'];
+}
+
+
+/**
+ * Get Hex color from RGB.
+ *
+ * @since n.e.x.t
+ *
+ * @param int $red Red 0-255.
+ * @param int $green Green 0-255.
+ * @param int $blue Blue 0-255.
+ *
+ * @return string|null Hex color or null if error.
+ */
+function get_hex_from_rgb( $red, $green, $blue ) {
```

Comment:
Let's prefix this function as usual. Also, I think it's more common for naming of this type of function to put the source format first, i.e. "rgb to hex". There are already some similarly named functions in WordPress core, e.g. `wp_tinycolor_hue_to_rgb()`. So I think we should rename this to:
```suggestion
function dominant_color_rgb_to_hex( $red, $green, $blue ) {
```